apps/omg_watcher/test/web/controllers/transaction_test.exs

```diff
@@ -260,47 +262,33 @@ defmodule OMG.Watcher.Web.Controller.TransactionTest do
            entity,
            limit \\ 200
          ) do
-      {:ok, _} = DB.Transaction.update_with(%{transactions: txs, blknum: 1_000, eth_height: 1})
```

Comment:
I think for the majority of tests we can base on initial-blocks fixture and do not need to insert data before each test. I’d consider to not use initial-blocks in special cases like testing endpoint on empty database or prepare specific test case that our fixture doesn’t support

Reply:
The idea of inserting data before each test was to have a readability of the tests:
1 insert something
2 check something 
In the first attempt I use ` initial-blocks ` but after our talk, we decide to switch to go with this.